packages/forms-web-app/src/dynamic-forms/dynamic-components/submission-screen/appellant.njk

```diff
@@ -4,6 +4,9 @@ Appeal Submitted - Appeal a planning decision - GOV.UK {% endblock %} {% block
 content %}
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+
+    TODO: FIX THIS
```

Comment:
still needs fixing

Reply:
I am not sure that we have updated copy for this

Reply:
I didn't add this as a comment though so will display to user!

Reply:
ah saw an old copy